vpdq/python/README.md

```diff
@@ -5,28 +5,41 @@ It is written in Cython.
 
 Development is hosted on [GitHub](https://github.com/facebook/ThreatExchange/tree/main/vpdq).
 
+Windows is currently not supported by the Cython bindings.
```

Comment:
I tried to get Windows working through MinGW. I actually got it to build with a few more changes that I wrote about [here](https://github.com/facebook/ThreatExchange/issues/1346#issuecomment-1682189882). But it gives errors about loading DLLs at runtime.

vpdq cpp implementation still builds and runs on Windows fine, though.

Reply:
That's fine for me